build/configs/qemu/scripts/flash-sdram-qemu-memory-increased.ld

```diff
@@ -114,6 +114,9 @@ SECTIONS
 		*(.gnu.linkonce.b.*)
 		*(COMMON)
 		_ebss = ABSOLUTE(.);
+		_sidle_stack = ABSOLUTE(.);
+		/* Heap start address in internal RAM */
+		_sint_heap_start = ABSOLUTE(.);
```

Comment:
You can change these lines as below, so that you can use  directly in the code. 
```
_sidle_stack = ABSOLUTE(.);
. = CONFIG_IDLETHREAD_STACKSIZE ;
_sint_heap_start = ABSOLUTE(.);
```